.github/workflows/main.yml

```diff
@@ -47,11 +47,19 @@ jobs:
         fetch-depth: ${{ matrix.fetch-depth }}
         submodules: true
 
+    - name: Restore cache
```

Comment:
This name is also used for the "post" action that is registered, so we now have a "Restore cache" and a "Post restore cache" that actually stores the cache. Most workflows just seem to use "Cache".